bn_mp_radix_size.c

```diff
@@ -4,33 +4,33 @@
 /* SPDX-License-Identifier: Unlicense */
 
 /* returns size of ASCII reprensentation */
-mp_err mp_radix_size(const mp_int *a, int radix, int *size)
+mp_err mp_radix_size(const mp_int *a, int radix, size_t *size)
```

Comment:
we currently can't change the signature of this function

Reply:
You wanted and said so?
But you're right, you didn't say when (Or did I miss it?) and it was in another context (`mp_count_bits` I think), so, yepp, my fault, will change it back.
And remove the superfluous "n", too, while I'm there ;-)

Are you willing to set a milestone for it?
LTM 2.0.0 maybe?